[ADD] Appointments: Basics doc (#12820)
odoo/documentation: content/applications/productivity/appointments.rst

- Select :guilabel:`Within a date range` to limit bookings to a specific range of dates. After
  selecting this option, click the :guilabel:`From` and :guilabel:`to` fields, and use the calendar
  pop-up to customize the date range.

Update the :guilabel:`Allow Cancelling` field to limit the amount of time before an appointment
where a customer can cancel. If this setting is enabled, customers are unable to cancel within the
designated time frame.

.. note::
   If a customer does try to cancel within the time frame, they receive an error message with
   contact information. If the appointment is for a resource, the contact details are for the user
   that created the appointment type. If the appointment is for a user, the contact details are for
   the user the appointment is with.

   .. image:: appointments/cancellation-message.png
      :alt: An example of the message a customer sees when cancelling.

Next, designate whether this appointment type is based on :guilabel:`Users` or
:guilabel:`Resources`, by selecting the appropriate radio button. If it is based on users, select
one or more :guilabel:`Users` in the drop-down. If it is based on :ref:`resources

> <productivity/appointments/resources>`, select one or more :guilabel:`Resources` in the drop-down.

Reviewer: When selecting 'Resource' there's an additional 'Manage Capacities' field you can check. Looks like if you check it, it'll limit it to the people it can handle, if you don't, then I guess it allows for unlimited people? Could make sense if a resource is a park, or open space, or a large room. if it's a table, then you'd need this checked. Just something to add.

   assigned a user/resource.

Schedule tab
------------

The :guilabel:`Schedule` tab is used to outline when this appointment type is to be made available.
The information listed here is used to create the time slots that appear on the booking page.

Click :guilabel:`Add a line` to create a new time frame. Select a day of the week from the
:guilabel:`Every` drop-down menu, then update the times in the :guilabel:`From` and :guilabel:`To`
fields. Click the :icon:`fa-trash-o` :guilabel:`(trash)` icon to delete an entry. Multiple entries
can be included for a single day.

.. tip::
   If an appointment should not be available at specific times, such as when users are taking lunch,
   include time slots before and after.

   .. image:: appointments/schedule-tab.png
      :alt: An example of the Schedule tab in an appointment.

Options tab
-----------

The :guilabel:`Options` tab is used to customize the display options for this appointment, as well
as notification settings for customers and users.

The :guilabel:`Front-End Display` field determines how the appointment is presented on the website

> to customers. Select the :guilabel:`Show Pictures` radio button to publish the default pictures of

Reviewer: The 'Website' field isn't listed here

the user or resources for this appointment on the website.

The :guilabel:`Timezone` and :guilabel:`Location` fields automatically populate for resource

> appointments, based on where the resource is location. For user based appointments, the

Reviewer: ```suggestion
appointments, based on where the resource is located. For user-based appointments, the
```

> The :guilabel:`Timezone` and :guilabel:`Location` fields automatically populate for resource
> appointments, based on where the resource is location. For user based appointments, the

Reviewer: ```suggestion
The :guilabel:`Timezone` and :guilabel:`Location` fields automatically populate for resource-based
appointments, based on where the resource is location. For user based appointments, the
```

:guilabel:`Location` field defaults to an `Online Meeting`, with a :guilabel:`Videoconference Lnik`
automatically generated. If this should not be an online meeting, select a different option in the
:guilabel:`Location` field.

Tick the :guilabel:`Manual Confirmation` checkbox to require approval before a meeting is accepted.
If this feature is enabled, the appointment time slot is still considered *reserved* until it is
confirmed or rejected. Leave this checkbox blank to automatically accept meetings created from this
appointment.

> 

Reviewer: There's also an 'Up-front Payment' field here

> The :guilabel:`Create Opportunities` feature adds a lead or opportunity to the **CRM** for each
> scheduled appointment, which is assigned to the responsible user. Tick the :guilabel:`Create

Reviewer: ```suggestion
The :guilabel:`Create Opportunities` feature adds a lead or opportunity to the **CRM** app for each
scheduled appointment, which is assigned to the responsible user. Tick the :guilabel:`Create
```

Opportunities` checkbox to enable this option.

.. important::
   This field is only visible if the **CRM** app is installed on the database.

> The :guilabel:`Reminders` field is used to set how customers are to be contacted before the
> appointment time. Select one or more options from the drop-down, based on both communication method,
> and the time frame.

Reviewer: ```suggestion
The :guilabel:`Reminders` field is used to set how customers are contacted before the
appointment time. Select one or more options from the drop-down, based on both communication methods,
and the time frame.
```

> 

Reviewer: There's also a 'Booking Emaik' and 'Cancellation Email' template fields, and a 'CC to' field

Tick the :guilabel:`Allow Guests` checkbox to grant customers the ability to add additional guests
when registering for an appointment.

Questions tab
-------------

The :guilabel:`Questions` tab can be used to prompt customers for additional information while they
are booking an appointment. Click :guilabel:`Add a line` to add a new question.

> On the :guilabel:`Create Questions` pop-up, enter the :guilabel:`Question`, then choose an

Reviewer: ```suggestion
On the :guilabel:`Create Questions` pop-up window, enter the :guilabel:`Question`, then choose an
```

:guilabel:`Answer Type`.

> 

Reviewer: There's also a 'Placeholder' field if you chose one of the text lines (you can mention that's where the preview/sample answer would be)

Tick the :guilabel:`Mandatory Answer` checkbox to require customers to answer this question before

> they are allowed to book an appointment. Click :guilabel:`Save & Close` when finished.
> 

Reviewer: ```suggestion
they are allowed to book an appointment.  Click :guilabel:`Save & New` to add another question, or :guilabel:`Save & Close` when finished.

```

Click :guilabel:`Add a line` to create a new time frame. Select a day of the week from the
:guilabel:`Every` drop-down menu, then update the times in the :guilabel:`From` and :guilabel:`To`
fields. Click the :icon:`fa-trash-o` :guilabel:`(trash)` icon to delete an entry. Multiple entries
can be included for a single day.

.. tip::
   If an appointment should not be available at specific times, such as when users are taking lunch,
   include time slots before and after.

   .. image:: appointments/schedule-tab.png
      :alt: An example of the Schedule tab in an appointment.

Options tab
-----------

The :guilabel:`Options` tab is used to customize the display options for this appointment, as well
as notification settings for customers and users.

The :guilabel:`Front-End Display` field determines how the appointment is presented on the website
to customers. Select the :guilabel:`Show Pictures` radio button to publish the default pictures of
the user or resources for this appointment on the website.

The :guilabel:`Timezone` and :guilabel:`Location` fields automatically populate for resource
appointments, based on where the resource is located. For user-based appointments, the

> :guilabel:`Location` field defaults to an `Online Meeting`, with a :guilabel:`Videoconference Lnik`

Reviewer 2: ```suggestion
:guilabel:`Location` field defaults to an `Online Meeting`, with a :guilabel:`Videoconference Link`
```

scheduled appointment, which is assigned to the responsible user. Tick the :guilabel:`Create
Opportunities` checkbox to enable this option.

.. important::
   This field is only visible if the **CRM** app is installed on the database.

The :guilabel:`Reminders` field is used to set how customers are to be contacted before the
appointment time. Select one or more options from the drop-down, based on the communication method,
and the time frame.

Tick the :guilabel:`Allow Guests` checkbox to grant customers the ability to add additional guests
when registering for an appointment.

Questions tab
-------------

The :guilabel:`Questions` tab can be used to prompt customers for additional information while they
are booking an appointment. Click :guilabel:`Add a line` to add a new question.

On the :guilabel:`Create Questions` pop-up window, enter the :guilabel:`Question`, then choose an
:guilabel:`Answer Type`.

Tick the :guilabel:`Mandatory Answer` checkbox to require customers to answer this question before
they are allowed to book an appointment. Click :guilabel:`Save & New` to add another question, or
:guilabel:`Save & Close` when finished.

Messages tab
------------

> The :guilabel:`Messages` tab is used to provide additional information regarding this appointment

Reviewer 2: ```suggestion
The :guilabel:`Messages` tab is used by the business to provide additional information to customers regarding this appointment
```

============
Appointments
============

Odoo's **Appointments** app is a self-service scheduling app that simplifies the process of booking
meetings, consultations, or services. Integrated with Odoo's suite of business apps, it allows
companies to automate appointment scheduling, reduce manual coordination, and provide a seamless
experience for clients. Appointments can be linked to calendars, **CRM** opportunities, employee
schedules, and more, making it an ideal tool for service-based businesses seeking efficiency and
organization.

Configuration
=============

The **Appointments** app allows for new appointments to be scheduled based on the availability of
users, or the availability of *resources*, such as meeting rooms or seating areas. To create a new
resource, or manage existing resources, navigate to :menuselection:`Appointments --> Configuration
--> Resources`. This opens a list of the available resources in the database, as well as their
individual capacity.

> .. _productivity/appointments/resources:

Reviewer 3: Nitpick– the custom anchor convention is to start with the app or module name, not the parent folder :)
```suggestion
.. _appointments/resources:
```
